nemoguardrails/actions/llm/utils.py

```diff
@@ -574,4 +576,6 @@ def escape_flow_name(name: str) -> str:
         .replace('"', "")
```

Comment:
I think the regex substitution below can replace these four string replacement statements.

Reply:
Yes, I agree. I'll remove those lines.